src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -568,8 +568,6 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
             }
         }
 
-        [ActiveIssue(23769)]
-        [ActiveIssue(22707, TestPlatforms.AnyUnix)]
```

Comment:
22707 was closed as a dup of 9228, which looks to be noted as tracking intermittent failures.  We can re-enable it, but I wouldn't be surprised if it fails again.

Reply:
Yes, lots of failures for related to #22707 (i.e.: #9228). I'm going to play with some test runs for this since it seems that at least 23769 is passing on Windows (there are other Outerloop failures for Windows).